packages/server/modules/fileuploads/rest/models.ts

```diff
@@ -0,0 +1,109 @@
+import { getModelByIdFactory } from '@/modules/core/repositories/models'
+import { getStreamFactory } from '@/modules/core/repositories/streams'
+import { saveUploadFileFactory } from '@/modules/fileuploads/repositories/fileUploads'
+import { createUploadFileFactory } from '@/modules/fileuploads/services/management'
+import { getProjectDbClient } from '@/modules/multiregion/utils/dbSelector'
+import { streamWritePermissionsPipelineFactory } from '@/modules/shared/authz'
+import { getPort } from '@/modules/shared/helpers/envHelper'
+import { authMiddlewareCreator } from '@/modules/shared/middleware'
+import { getRolesFactory } from '@/modules/shared/repositories/roles'
+import { publish } from '@/modules/shared/utils/subscriptions'
+import type { Express } from 'express'
+import { Knex } from 'knex'
+import request from 'request'
+
+export const modelsRouter = (app: Express, db: Knex): void => {
+  app.post(
+    '/api/file/:fileType/projects/:projectId/models/:modelId',
```

Comment:
`api/projects/:projectId/models/:modelId/fileUpload` should be the new route

fileType can be droppped